website/docs/configuration/overview.html.md

```diff
@@ -0,0 +1,182 @@
+---
+layout: "language"
+page_title: "Configuration Language"
+sidebar_current: "docs-config-index"
+description: |-
+  Terraform uses text files to describe infrastructure and to set variables.
+  These text files are called Terraform _configurations_ and are
+  written in the Terraform language.
+---
+
+# Configuration Language
```

Comment:
I'm not sure I really understand why the H2 headings in this page are organized the way they are 🤔 It might be nice to start with the big picture (folder structure) and then zoom in to the small stuff (syntax). That's just one suggestion, I'm sure there are other ways to order the info that might work just as well.